lib/chef/resource/chef_role.rb

```diff
@@ -2,103 +2,107 @@
 require 'chef/resource/lwrp_base'
 require 'chef/run_list/run_list_item'
 
-class Chef::Resource::ChefRole < Chef::Resource::LWRPBase
-  self.resource_name = 'chef_role'
+class Chef
+  class Resource
```

Comment:
uhmm that was the test one and I leaved like that.. I guess I need to change it..